src/vs/platform/terminal/test/node/terminalEnvironment.test.ts

```diff
@@ -163,23 +162,6 @@ suite('platform - terminalEnvironment', () => {
 							assertIsEnabled(result1);
 						});
 					});
-					suite('should incorporate global USER_ZDOTDIR env variable', () => {
```

Comment:
we should not rely upon `USER_ZDOTDIR` being set. instead, we'll make sure that `ZDOTDIR` is unset for both login and non-login `zsh`